ci/ci/github.py

```diff
@@ -51,6 +51,24 @@ class GithubStatus(Enum):
     FAILURE = 'failure'
 
 
+def github_status(state: str) -> GithubStatus:
+    """
+    Converts a state for a commit status (https://docs.github.com/en/graphql/reference/enums#statusstate)
+    or a conclusion for a check (https://docs.github.com/en/graphql/reference/enums#checkconclusionstate)
+    from the GraphQL API to a GithubStatus.
+    """
+    _state = state
+    if state in {"EXPECTED", "ACTION_REQUIRED", "STALE"}:
+        _state = "PENDING"
+    elif state in {"ERROR", "TIMED_OUT", "CANCELLED", "STARTUP_FAILURE"}:
+        _state = "FAILURE"
+    elif state in {"NEUTRAL", "SKIPPED"}:
```

Comment:
Hmm, SKIPPED for a required check being counted as SUCCESS feels a little wrong to me. Should this be PENDING?